backend/src/utils/v2/entity.ts

```diff
@@ -1,4 +1,10 @@
-export function toEntity(kind: string, value: string) {
+export const EntityKind = {
+  User: 'user',
+  Group: 'group',
```

Comment:
I don't think this can be an enum, necessarily.  Systems can generate their own entity kinds, e.g. we'll have ones specific to our internal names.